From 1f1c844bfc9bac8e8c840a451f6bd0b7ca2c9e47 Mon Sep 17 00:00:00 2001
From: Nadav Haklai <nadavh@marvell.com>
Date: Thu, 23 Aug 2012 14:41:07 +0300
Subject: [PATCH 264/609] Add CONFIG_IKCONFIG_PROC to defconfigs - allow
 reading the .config file from /proc/config.gz at
 runtime

Signed-off-by: Seif Mazareeb <seif@marvell.com>
---
 arch/arm/configs/armada_xp_amp_g0_defconfig |    2 ++
 arch/arm/configs/armada_xp_amp_g1_defconfig |    2 ++
 arch/arm/configs/armada_xp_defconfig        |    2 ++
 arch/arm/configs/armada_xp_nas_defconfig    |    2 ++
 4 files changed, 8 insertions(+)

diff --git a/arch/arm/configs/armada_xp_amp_g0_defconfig b/arch/arm/configs/armada_xp_amp_g0_defconfig
index 549513d..625f6bb 100644
--- a/arch/arm/configs/armada_xp_amp_g0_defconfig
+++ b/arch/arm/configs/armada_xp_amp_g0_defconfig
@@ -2,6 +2,8 @@
 CONFIG_EXPERIMENTAL=y
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 CONFIG_SYSCTL_SYSCALL=y
diff --git a/arch/arm/configs/armada_xp_amp_g1_defconfig b/arch/arm/configs/armada_xp_amp_g1_defconfig
index 26b739a..21c5066 100644
--- a/arch/arm/configs/armada_xp_amp_g1_defconfig
+++ b/arch/arm/configs/armada_xp_amp_g1_defconfig
@@ -2,6 +2,8 @@
 CONFIG_EXPERIMENTAL=y
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 CONFIG_SYSCTL_SYSCALL=y
diff --git a/arch/arm/configs/armada_xp_defconfig b/arch/arm/configs/armada_xp_defconfig
index c9b189f..d25c3b2 100644
--- a/arch/arm/configs/armada_xp_defconfig
+++ b/arch/arm/configs/armada_xp_defconfig
@@ -2,6 +2,8 @@
 CONFIG_EXPERIMENTAL=y
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 CONFIG_SYSCTL_SYSCALL=y
diff --git a/arch/arm/configs/armada_xp_nas_defconfig b/arch/arm/configs/armada_xp_nas_defconfig
index adeb451..33782bb 100755
--- a/arch/arm/configs/armada_xp_nas_defconfig
+++ b/arch/arm/configs/armada_xp_nas_defconfig
@@ -2,6 +2,8 @@
 CONFIG_EXPERIMENTAL=y
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 CONFIG_SYSCTL_SYSCALL=y
-- 
1.7.9.5

